pkg/fakerp/config.go

```diff
@@ -87,16 +88,17 @@ func getPluginConfig() (*api.PluginConfig, error) {
 	}
 
 	// populate geneva artifacts
-	artifactDir := "secrets/"
-	logCert, err := readCert(artifactDir + "logging-int.cert")
+	gp := os.Getenv("GOPATH")
```

Comment:
i think this could work but more care needs to be taken in the code for the case where the system has multiple `GOPATH` directories configured

Reply:
I think good practice is to have to have one GOPATH, or just switch it between projects if you really need?